<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ShowMoreInfo</class>
 <widget class="QDialog" name="ShowMoreInfo">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>334</width>
    <height>531</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>334</width>
    <height>531</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>334</width>
    <height>531</height>
   </size>
  </property>
  <property name="font">
   <font>
    <family>微软雅黑</family>
    <pointsize>10</pointsize>
   </font>
  </property>
  <property name="windowTitle">
   <string>更多信息</string>
  </property>
  <property name="windowIcon">
   <iconset resource="TimeTableQt.qrc">
    <normaloff>:/TimeTableQt/123.ico</normaloff>:/TimeTableQt/123.ico</iconset>
  </property>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>311</width>
     <height>511</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QListView" name="listView"/>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QComboBox" name="comboBox">
        <property name="currentText">
         <string>Mon</string>
        </property>
        <item>
         <property name="text">
          <string>Mon</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Tue</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Wed</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Thu</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Fri</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Sat</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Sun</string>
         </property>
        </item>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="lineEdit"/>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <item>
       <widget class="QPushButton" name="pushButton">
        <property name="text">
         <string>添加</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="pushButton_2">
        <property name="text">
         <string>关闭</string>
        </property>
        <property name="default">
         <bool>true</bool>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
 </widget>
 <resources>
  <include location="TimeTableQt.qrc"/>
 </resources>
 <connections/>
</ui>
